src/components/routes/Host/ListingForm/ListingForm.tsx

```diff
@@ -192,7 +192,6 @@ class ListingForm extends React.Component<Props, State> {
                     onClick={() => {
                       if (!FormikProps.isValid) {
                         alert(`Cannot save changes due to errors:\n\n${Object.values(FormikProps.errors).join('\n').toString()}`);
```

Comment:
might need to change`\n` to `\n\r` fo windows

Reply:
so with just using `\n` it looks like:

![image](https://user-images.githubusercontent.com/18321302/52228771-1bd27d00-2868-11e9-8084-bcec8279ee4b.png)

but with `\n\r` it looks like there are double spaces:

![image](https://user-images.githubusercontent.com/18321302/52228807-2e4cb680-2868-11e9-8f3b-3dd6478a6477.png)